gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=10.3.1
+version=11.0.0
```

Comment:
Creating a new major version due to the refactor needed to support multiple resource types in the submit provider return response.